src/mercury_bulk.h

```diff
@@ -200,7 +194,7 @@ HG_Bulk_get_segment_count(hg_bulk_t handle);
  *
  * \return Non-negative value
  */
-static HG_INLINE uint8_t
+static HG_INLINE uint32_t
```

Comment:
can you please keep it as uint8_t and fix HG_Bulk_get_flags() to instead return `((struct hg_bulk_desc_info *) handle)->flags & 0x3`